content/updates/2020-10/5-DQI-new-missing-image-criteria.md

```diff
@@ -8,4 +8,10 @@ pim_announcement_audience:
 ::: meta-data type="New" features="Data quality" available="late October" link-to-doc="../articles/.html"
 :::
 
-TODO
+As a picture is worth a thousand words and are really crucial if you want to ensure an optimal product experience, Data Quality Insights now checks whether your products have an image or not. Don’t forget images, it will decrease your enrichment grade!
```

Comment:
```suggestion
You all know the expression _a picture is worth a thousand words_ don't you? Images are a critical element in designing compelling product experiences. Data Quality Insights now checks whether your products have an image or not. Don’t forget images and assets, otherwise, it will decrease your enrichment grade!
```

Reply:
What's new is far from being in the first part of the paragraph. I think it will render badly in the announcement panel, as customers won't understand what's new. I would suggest:
```suggestion
You all know the expression _"A picture is worth a thousand words"_, don't you? Data Quality Insights now checks whether your products have an image or not. Images are a critical element in designing compelling product experiences. Don’t forget them, otherwise, it will decrease your enrichment grade!
```

Reply:
I would not split the first 2 sentences about images and PX, they go together. I've removed the last part 'don't forget..' this is unnecessary imho. The title will help them to understand what's new. 
Also would not put a capital A to "A picture..", and either we put the "" or the italic, we can't put both.
```suggestion
You all know the expression _a picture is worth a thousand words_ don't you? Images are a critical element in designing compelling product experiences. Data Quality Insights now checks whether your products have an image or not. 
```
Maybe you're right about the render, but I'd like to try it, if not ok we'll change it.

Reply:
Ok, let's do this. But let's keep the last sentence in another paragraph as I feel it's important to say that this new criteria as an impact on the enrichment grade.
```suggestion
You all know the expression _"A picture is worth a thousand words"_, don't you?  Images are a critical element in designing compelling product experiences. Data Quality Insights now checks whether your products have an image or not. 

Don’t forget them, otherwise, it will decrease your enrichment grade!
```